tools/test_containers_compose

```diff
@@ -5,8 +5,8 @@ set -euo pipefail
 for i in webui worker; do
   (
     cd container/$i/ &&
-    sudo docker-compose build --parallel &&
```

Comment:
with the removal of `--parallel', what is the impact?

Reply:
This flag is being ignored
`Flag '--parallel' is ignored when building with COMPOSE_DOCKER_CLI_BUILD=1`

For instance, see this run
https://github.com/os-autoinst/openQA/runs/2342931733